machine_configuration/machine-config-pin-preload-images-about.adoc

```diff
@@ -0,0 +1,33 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="machine-config-pin-preload-images-about"]
+= About pinning and preloading images
+include::_attributes/common-attributes.adoc[]
+:context: machine-config-operator
+
+toc::[]
+
+Slow, unreliable connection to an image registry can interfere with operations that require pulling images, such as updating a cluster or deploying an application. This can include clusters that have low bandwidth, clusters with unreliable internet connectivity, or clusters in a disconnected environment. 
+
+For example, a cluster update might require pulling more than one hundred images. Failure to pull those images could cause retries that can interfere with the update process and might cause the update to fail. One way to prevent this is to _preload_ the required images by pulling them in advance, before they are actually needed. Then, by _pinning_ them to a machine config pool, you ensure that the images are available when needed. Pinning and preloading images can provide a more consistent update, which is important when scheduling updates into maintenance windows.
+
+Pinning and preloading images also ensures that the images are available when deploying applications, which allows you to deploy in a predictable time. Another benefit to pinning and preloading images is that image garbage collection does not remove the pinned images.
+
+You can pin and preload images by using a a `PinnedImageSet` custom resource (CR) as described in the following section. Pinned images are stored on the nodes in the `/etc/crio/crio.conf.d/50-pinned-images` file. The contents of the file appear similar to the following example:
```

Comment:
```suggestion
You can preload and pin images by using a a `PinnedImageSet` custom resource (CR). Pinned images are stored on the nodes in the `/etc/crio/crio.conf.d/50-pinned-images` file. The contents of the file are similar to the following example:
```
I removed "as described in the following section" because I wasn't sure whether you were referring to the example immediately after this paragraph or the procedure module. Either way, I think it's clear enough that the information that follows this paragraph further explains how to preload and pin images.